### ISSUE TITLE
bug: ORACLE plugin: no data in oracle_virtualnetwork_private_ips resource table

### ISSUE THREAD

**Issue by gianluca-ava** [cloudquery/cloudquery#18116]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The resource table oracle_virtualnetwork_private_ips doesn't show any data.

### Expected Behavior

oracle_virtualnetwork_private_ips must show the created private ips.

### CloudQuery (redacted) config

```kind: source
spec:
  # Source spec section
  name: drcc
  path: "cloudquery/oracle"
  version: "v6.3.0"
  tables: ["*"]
  skip_tables:
    - "oracle_compute_images"
    - "oracle_compute_vnic_attachments"
    - "oracle_virtualnetwork_fast_connect_provider_services"
    - "oracle_networkfirewall_network_firewall_policies"
    - "oracle_networkfirewall_network_firewalls"
    - "oracle_networkfirewall_work_requests"
    - "oracle_networkloadbalancer_work_requests"
  destinations: ["postgresql"] 
  spec:
    concurrency: 500
---
kind: destination
spec:
  name: "postgresql"
  path: "cloudquery/postgresql"
  version: "v8.0.6"
  spec:
    write_mode: "overwrite-delete-stale"
    connection_string: "postgresql://gamadm:${password}@${host}:${port}/${database}?sslmode=require"
```

### CloudQuery (redacted) logs

```{
    "level": "error",
    "module": "oracle-src",
    "invocation-id": "faafb732-8858-4358-9a60-0d2705cd68d2",
    "client": "xxxx",
    "error": "Error returned by VirtualNetwork Service. Http Status Code: 400. Error Code: MissingParameter. Opc request id: 43f6f84bd4aa2f330b1bfad867191a1f/44AC646527C349353DED99D38E6C24F4/06BCAC35F656A7D7A7AFBDBCA43C4E48. Message: Missing Parameter\nOperation Name: ListPrivateIps\nTimestamp: 2024-06-03 07:40:03 +0000 GMT\nClient Version: Oracle-GoSDK/65.28.3\nRequest Endpoint: GET https://iaas.eu-zurich-1.oraclecloud.com/20160918/privateIps?limit=100\nTroubleshooting Tips: See https://docs.oracle.com/iaas/Content/API/References/apierrors.htm#apierrors_400__400_missingparameter for more information about resolving this error.\nAlso see https://docs.oracle.com/iaas/api/#/en/iaas/20160918/PrivateIp/ListPrivateIps for details on this operation's requirements.\nTo get more info on the failing request, you can set OCI_GO_SDK_DEBUG env var to info or higher level to log the request/response details.\nIf you are unable to resolve this VirtualNetwork issue, please contact Oracle support and provide them this full error message.",
    "message": "table resolver finished with error",
    "table": "oracle_virtualnetwork_private_ips",
    "time": "2024-06-03T07:40:03Z"
}
```

### CloudQuery version

5.20.3

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by bbernays** [comment]
Based on the docs, this cannot be a top level table as the request requires one of the following:
```
Subnet OCID 
VNIC OCID 
Both private IP address and subnet OCID: This lets you get a `privateIP` object based on its private IP address (for example, 10.0.3.3) and not its OCID. For comparison, GetPrivateIp requires the OCID
```

https://pkg.go.dev/github.com/oracle/oci-go-sdk/v65@v65.65.1/core#VirtualNetworkClient.ListPrivateIps

**Comment by gianluca-ava** [comment]
Hi @bbernays, thanks for the answer. Does it mean that the table will be always empty, right?

**Comment by savme** [comment]
Hi @gianluca-ava! Thanks for the report!

We’ll need to look into making this table dependent on another table that provides the required parameters. I created an issue for that, but there’s no specific timeline for it yet. We'll keep you posted on the progress

**Comment by savme** [comment]
@gianluca-ava a fix for this issue has been released in [v7.0.0](https://hub.cloudquery.io/plugins/source/cloudquery/oracle/v7.0.0). Please note that you can now find the private_ip data in the following tables: `oracle_virtualnetwork_subnet_private_ips` and `oracle_compute_vnic_private_ips`.

**Comment by savme** [comment]
Closed as fixed. Please comment if you have any other issues with `private_ips`.
